### DISCUSSION TITLE
推荐新音乐这个接口是取消了吗？一直报错404

### DISCUSSION THREAD

**Discussion by Three-deaths** [Binaryify/NeteaseCloudMusicApi#1745] [Q&A]
/personalized/newsong 
完整地址：https://netease-cloud-music-api-six-rust.vercel.app/personalized/newsong
我这里请求一直是404不知道为什么

**Comment by Binaryify** [comment]
这是你自己部署的吗

**Reply by Three-deaths** [reply]
这个项目初始模板是我在git上面下来的，里面那时候已经有这个接口在请求使用了，现在我发现接口请求一直报错404不存在，我在：https://neteasecloudmusicapi-docs.4everland.app/#/?id=%e6%8e%a8%e8%8d%90%e6%96%b0%e9%9f%b3%e4%b9%90
这里发现了这个接口写的文档，按照上面的要求修改了，仍然还是404报错，请问是我写的地址错了吗？

**Reply by Three-deaths** [reply]
![image](https://github.com/Binaryify/NeteaseCloudMusicApi/assets/95597842/2265f75d-a983-4931-96fa-89fc7f67c542)

**Comment by Binaryify** [comment]
无法复现
<img width="1765" alt="image" src="https://github.com/Binaryify/NeteaseCloudMusicApi/assets/12221718/6e873504-e887-48a4-ac16-0af85549e786">
